src/metabase/cmd/copy.clj

```diff
@@ -106,13 +106,11 @@
 (defn- insert-chunk!
   "Insert of `chunk` of rows into the target database table with `table-name`."
   [target-db-type target-db-conn table-name chunkk]
-  (print (color/blue \.))
-  (flush)
```

Comment:
This was a useful bit of debugging feedback, especially for very large dumps, as it provides some idea of "where" a dump may have failed. Could we maybe add a debug log here?

Reply:
The only issue there is that with `log/` we can't append to the current line... would logging a bunch of `.` lines still make sense?

Reply:
How about simply `(log/debug "Inserting chunk.")`? It'll be noisy, but at least we can get the data when necessary.

Reply:
it looks like we have `:mb-run-mode     "test"` available during test runs. could print those blue periods when this value is not set? Alternatively, if we've gotten the env var for cloud instances yet we could watch on that?

Reply:
I just added `Inserting chunk` like @walterl suggested 💀 

I think writing each message on a new line might actually be preferable if something somewhere is buffering the output and not displaying lines until the newline is written

Reply:
Yes please to that. I want to change from using `clojure.java.shell/sh` in Harbormaster to something that we can watch the output from in realtime, so this is exactly something I'd want from the output. Pretend there's no pty attached.

Reply:
@robdaemon you can definitely watch output in realtime from a shell process. Look at what I did here for the new build scripts. You just need to spin up another thread to read from the process input streams for stdout & stderror 

https://github.com/metabase/metabase/blob/master/bin/common/src/metabuild_common/shell.clj#L9-L67

Reply:
thanks! we're using https://clojuredocs.org/clojure.java.shell/sh right now, which does not allow that, but we'll move to exactly what the core app is doing soon! it'll make a difference in development / debugging, that's for sure